server/linter/plugins/standards/rules/ensure_span_naming_rule.go

```diff
@@ -77,18 +83,22 @@ func (r ensureSpanNamingRule) validateHTTPSpanName(ctx context.Context, span *mo
 }
 
 func (r ensureSpanNamingRule) validateDatabaseSpanName(ctx context.Context, span *model.Span) model.Result {
-	var expectedName string
 	dbOperation := span.Attributes.Get("db.operation")
-	dbName := span.Attributes.Get("db.operation")
+	dbName := span.Attributes.Get("db.name")
 	tableName := span.Attributes.Get("db.sql.table")
+	expectedName := fmt.Sprintf("%s %s.%s", dbOperation, dbName, tableName)
 
-	if tableName != "" {
-		expectedName = fmt.Sprintf("%s %s %s", dbOperation, dbName, tableName)
-	} else {
-		expectedName = fmt.Sprintf("%s %s", dbOperation, dbName)
+	// TODO: fix this by adding proper validation for all other db systems
+	dbSystem := span.Attributes.Get("db.system")
```

Comment:
For Redis, we have to match the prefix of the statement to validate the name. This should be improved in the future to run checks by db system